README.md

```diff
@@ -22,8 +22,8 @@ browse the [SMW wiki](https://www.semantic-mediawiki.org) for a more comprehensi
 
 ## Requirements
 
-- PHP 5.3 or later (5.5 or later recommended)
-- MediaWiki 1.19 or later (1.25 or later recommended)
+- PHP 5.5 or later (5.6 or later recommended)
+- MediaWiki 1.23 or later (1.27 or later recommended)
```

Comment:
@JeroenDeDauw I think this may be merged since we are approaching 2.5.0. At some point people just have to upgrade their software.